packages/app/src/runs/modals/CreateCloudOrgModal.vue

```diff
@@ -13,10 +13,9 @@
         <ExternalLink
           class="border rounded mx-auto border-gray-100 py-4px px-16px text-indigo-500 inline-block"
           :href="createOrgUrl"
-          :prefix-icon="OrganizationIcon"
           :include-graphql-port="true"
-          prefix-icon-class="icon-light-transparent icon-dark-white"
         >
+          <i-cy-office-building_x16 class="inline-block icon-dark-gray-500" />
```

Comment:
It looks like there's some overall differences in style between what's here and what's [in Figma](https://www.figma.com/file/PAOozuCxtfetqP8fUVvg0y/Cypress-App?node-id=4597%3A31174). Seeing the icon is an improvement, but it looks like we'd need to transition to a button to match the design.

Reply:
I was trying to look at the figma yesterday, but didn't have access until just now. I've updated the button style to match the design.

Reply:
I think a link is more accessible / semantic. it only takes a few styles to make it look like a button anyway.